### DISCUSSION TITLE
BlockHandler stops ticking

### DISCUSSION THREAD

**Discussion by powerbute** [Minestom/Minestom#1692] [Help] [answered]
Stops ticking if break the block from above, why can this happen?

`
public class ExampleBlock implements BlockHandler {`

    @Override
    public void tick(@NotNull Tick tick) {
        BlockHandler.super.tick(tick);
        System.out.println("TICK");
    }

    @Override
    public @NotNull NamespaceID getNamespaceId() {
        return null;
    }

    @Override
    public boolean isTickable() {
        return true;
    }
`}`

Main:
`if(event.getBlock().equals(Block.LANTERN)) {
                event.setBlock(Block.LANTERN.withHandler(new ExampleBlock()));
                System.out.println("ADD BLOCK HANDLER");
            }`

**Comment by powerbute** [comment, marked as answer]
https://github.com/ALS-Project/Minestom-Placement DONT USING IT!

The whole problem was using this, so removing this from the code helped me

**Comment by MrBretze** [comment]
Can you create a issue on the project please ?
And a simple demo to reproduce your issue ?

My extension/librairies doesn't no use BlockHandler
